templates/agenda/show.html.twig

```diff
@@ -0,0 +1,204 @@
+{% extends 'layout-with-agenda-submenu.html.twig' %}
+
+{% trans_default_domain 'agenda' %}
+
+{% block title %}{% trans %}Agenda{% endtrans %}{% endblock %}
+
+{% block header %}
+    <div class="row">
+        <div class="col">
+            <h1>{% trans %}Agenda{% endtrans %}</h1>
+        </div>
+        <div class="col-auto">
+            {% if 4 == agenda.status %}
+                <button class="btn btn-danger" disabled data-toggle="tooltip" data-placement="auto" title="{% trans %}Cannot delete finished agenda{% endtrans %}">{% trans %}Delete agenda{% endtrans %}</button>
+            {% elseif agenda.agendaItems|length > 0 %}
+                <button class="btn btn-danger" disabled data-toggle="tooltip" data-placement="auto" title="{% trans %}Cannot delete agenda containing agenda items{% endtrans %}">{% trans %}Delete agenda{% endtrans %}</button>
+            {% else %}
+                <form id="myform" class="btn" method="post" action="{{ path('agenda_delete', {'id': agenda.id}) }}" onsubmit="return confirm('{% trans %}Are you sure you want to delete this agenda?{% endtrans %}');">
+                    <input type="hidden" name="_method" value="DELETE">
+                    <input type="hidden" name="_token" value="{{ csrf_token('delete' ~ agenda.id) }}">
+                    <button class="btn btn-danger" data-toggle="tooltip" data-placement="auto" title="{% trans %}Delete agenda{% endtrans %}">{% trans %}Delete agenda{% endtrans %}</button>
+                </form>
+            {% endif %}
+            {% if 3 == agenda.status %}
```

Comment:
`3` should be replaced by a constant or method on the agenda class.

Reply:
Added and used `isReady` function to `Agenda` class.